### DISCUSSION TITLE
Error while making Transaction: Transaction simulation failed: This account may not be used to pay transaction fees

### DISCUSSION THREAD

**Discussion by dhruvja** [espresso-cash/espresso-cash-public#565] [Q&A] [answered]
I am trying to build a transaction to call the smart contract written using anchor. When i create a new keypair and make a transaction, it goes through. But when i use the same keypair and try to make another transaction, it fails with the following error
` Transaction simulation failed: This account may not be used to pay transaction fees`.
Any transaction after that fails with the above error.

Here is the snippet of the code I am using to build the transaction.
```dart
            const programId = "G33cFQAANWSrECutfYThVxiLYNLyZNAUcESbYzRwge3M";
            const createAccountInstructionIndex = [
              126,
              54,
              85,
              33,
              226,
              32,
              195,
              32
            ];
            final instruction = Instruction(
                accounts: [
            AccountMeta(
                pubKey: account.address,
                isWriteable: true,
                isSigner: false),
            AccountMeta(
                pubKey: account.address,
                isWriteable: true,
                isSigner: true),
            AccountMeta(
                pubKey: SystemProgram.programId,
                isWriteable:        //     isSigner: false)
                ],
                programId: programId,
                data: Buffer.fromConcatenatedByteArrays([
                  createAccountInstructionIndex,
                ]));

            final message = Message(
              instructions: [instruction],
            );

            final signature = await account.client.rpcClient
                .signAndSendTransaction(message, [account.wallet]);
```
The signer has more than enough funds to pay for tx.
Please tell me where am i going wrong.

**Comment by ookami-kb** [comment, marked as answer]
I guess this is some custom program. If it sets ownership over the account, it cannot be used for paying the fees. Your problem looks similar to this question: https://stackoverflow.com/questions/69779501/is-it-possible-to-corrupt-solana-smart-contract-from-client-side
